draft-ietf-rats-corim.md

```diff
@@ -1764,11 +1776,39 @@ Reference Value does not match.
 
 ##### Comparison for raw-value entries
 
-> I think this comparison method only works if the entry is at key 4 (because
-there needs to be a mask at key 5). Should we have a Reference Value of this
-which stores [expect-raw-value raw-value-mask] in an array?
+The value stored under `measurement-values-map` key 4 is a raw-value entry with
+optional mask. The mask, if present, is stored under `measurement-values-map` key 5.
+Both raw-value and mask must have type BSTR.
 
-[^issue]: Content missing. Tracked at https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/71
+Within a Reference Value, a `measurement-values-map` entry of type BSTR, either with
+no CBOR tag or with CBOR tag #6.560, is a raw-value entry without mask.
+
+If no mask is provided then the Verifier creates a mask with every byte containing
+the value 0xFF.
+If the Reference Value is not tagged then the created mask is the same size as
+the longer of Reference Value raw-value or Accepted Claims Set raw-value.
+If the Reference Value is tagged with #6.560 then the created mask is the same
+size as the Reference Value raw-value.
+
+For example, if the Accepted Claims Set raw-value has value h'123456',
+the untagged Reference Value raw-value has value h'1234', and there is no
+corresponding mask then a mask is created with value h'FFFFFF'.
+
+The Verifier then adds zeros to the end of each of the three BSTR inputs as required to make them all the same length.
+
+For example, if the Reference raw-value has value h'1234' and the Accepted Claims Set raw-value is h'123456' then the Reference raw-value is extended to h'123400'.
+
+Finally, the contents of the Reference Value and Accepted Claims Set raw-values are compared together using the mask. Each bit is compared separately.
+If a bit in the mask is 1 and the corresponding bits in the Reference Value and
+Accepted Claims Set are different, then the Reference Value does not match.
+If a bit in the mask is 0 then the values of the corresponding bits in the
+Reference Value and Accepted Claims set do not affect the result of the comparison.
+
+The CBOR tag (or lack of tag) on the Accepted Claims Set raw-value does not affect
+the result of the comparison.
+
+If a Reference Value `measurement-values-map` contains key 5 but does not contain
+key 4 then this is an error and the Reference Value does not match.
```

Comment:
```suggestion

```

Reply:
The cddl `? ( <raw value group stuff> )` ensures that if a `raw-value` is omitted, so will the `raw-value-mask` be omitted.